spec/datadog/appsec/processor/context_spec.rb

```diff
@@ -255,6 +255,23 @@
         context.run(input, timeout)
       end
     end
+
+    context 'run failed with libddwaf low-level exception' do
+      before do
+        allow(context.instance_variable_get(:@context)).to receive(:run).with(input, timeout)
+          .and_raise(Datadog::AppSec::WAF::LibDDWAF::Error, 'Could not convert persistent data')
+      end
+
+      let(:result) { context.run(input, timeout) }
+
+      it 'sends telemetry report' do
+        expect(telemetry).to receive(:error).with(/libddwaf:[\d.]+ execution error: :err_internal/)
+        expect(telemetry).to receive(:report)
```

Comment:
This suppose to be very unfortunate case, we will start with this approach and refactor it next (to report once?)